backend/src/bundles/payment/helpers/get-card-expire-date.ts

```diff
@@ -0,0 +1,8 @@
+import { type Stripe } from 'stripe';
+
+const getCardExpireDate = (card: Stripe.PaymentMethod.Card): Date => {
+    const { exp_month, exp_year } = card;
```

Comment:
Names of variables should be written in camelCase notation.

Reply:
it is from stripe card